README.md

```diff
@@ -5,9 +5,10 @@ A MMO based on pong game's concepts
 ## Getting started
 ### Locally
 1. `git clone` the repo
-2. Start the server : `npm run server`
-3. Start the client : `npm run client`
-4. Go to http://localhost:3000
+1. Statt datanase : `npm run db`
```

Comment:
We are using yarn... so commands must be : 
- `yarn db`
- `yarn server`
- `yarn client`